widgets/bat_openbsd.lua

```diff
@@ -0,0 +1,74 @@
+-- {{{ Grab environment
+local setmetatable = setmetatable
+local tonumber = tonumber
+local string = { format = string.format, gmatch = string.gmatch }
+local io = { popen = io.popen }
+local math = { floor = math.floor }
+local helpers = require("vicious.helpers")
+local string = {
+    gmatch = string.gmatch,
+    match = string.match,
+    format = string.format
+
+}
+-- }}}
+local bat_openbsd = {}
+
+local function worker(format, warg)
+    local battery = warg or "bat0"
+    local filter = string.format("hw.sensors.acpi%s", battery)
+    local cmd = string.format("sysctl -a | grep '^%s'", filter)
```

Comment:
I acknowledge that this is totally up to personal preference but maybe we can join these two lines?

Reply:
Since I reuse the filter variable later in the function, I'd rather keep it. I can get rid of the `cmd` one if you want but, IMHO, that function, as it is, is pretty easy to follow.

Reply:
> I reuse the filter variable later in the function.

I'm so sorry that I missed that. Yes, please keep it that way.